### ISSUE TITLE
use my dataset to train FCOS net，why the loss is nan?

### ISSUE THREAD

**Issue by funny000** [tianzhi0549/FCOS#381]
my dataset is remote dataset DIOR that come from Northwestern Polytechnical University, so when i use the dataset to train FCOS net, the loss is nan,

who can ask the reason?

the follow is error informatoin:
==================================================
2022-06-17 07:54:53.057 | INFO     | __main__:train:229 - cls loss : 5515.9775390625, reg loss : 7.476679801940918, ness loss : 0.7106261253356934, sum loss : 279.5187072753906
2022-06-17 07:54:54.175 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:54:55.242 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:54:56.419 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:54:57.572 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:54:58.608 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:54:59.634 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:00.939 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:02.146 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:03.483 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:04.700 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:05.925 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:06.957 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:08.256 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:09.364 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:10.461 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:11.527 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:12.584 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:13.665 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:14.936 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
2022-06-17 07:55:16.000 | INFO     | __main__:train:229 - cls loss : nan, reg loss : nan, ness loss : nan, sum loss : nan
=======================================================

**Comment by funny000** [comment]
use torch.nn.utils.clip_nn_norm_ function can solve the problem and add the a small learning rate
